README.markdown

```diff
@@ -279,12 +280,12 @@ else {
 
 * There should be exactly one blank line between methods to aid in visual clarity and organization. Whitespace within methods should separate functionality, but having too many sections in a method often means you should refactor into several methods.
 
-* Colons always have no space on the left and one space on the right. Exceptions are the ternary operator `? :`, empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
+* Colons always have no space on the left and one space on the right. Exceptions are the ternary operator `? :`, dictionaries `["A" : 1.2]`, empty dictionary `[:]` and `#selector` syntax for unnamed parameters `(_:)`.
```

Comment:
I believe dictionaries still follow `["key": "value"]` with no space on the left, and one space on the right. It's only the empty dictionary `[:]` that carries no spaces around